ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -231,97 +231,118 @@ Item {
             }
         }
 
-        StatusChatInput {
-            id: chatInput
-
+        Item {
             Layout.fillWidth: true
             Layout.margins: Style.current.smallPadding
+            Layout.preferredHeight: chatInput.height
+            StatusChatInput {
+                id: chatInput
+                width: parent.width
+                enabled: (root.rootStore.sectionDetails.joined && !root.rootStore.sectionDetails.amIBanned &&
+                          root.rootStore.isUserAllowedToSendMessage) && !tokenHoldRestrictions.visible
+
+                store: root.rootStore
+                usersStore: d.activeUsersStore
+
+                textInput.placeholderText: {
+                    if (enabled) {
```

Comment:
```suggestion
                    if (!enabled) {
```

Reply:
true, I'll change it to !tokenHoldRestrictions.visible to be more specific.